src/test/java/com/puppycrawl/tools/checkstyle/checks/blocks/AvoidNestedBlocksCheckTest.java

```diff
@@ -57,7 +57,7 @@ public void testStrictSettings()
             "50:17: " + getCheckMessage(MSG_KEY_BLOCK_NESTED),
             "58:17: " + getCheckMessage(MSG_KEY_BLOCK_NESTED),
         };
-        verify(checkConfig, getPath("InputNestedBlocks.java"), expected);
+        verify(checkConfig, getPath("InputAvoidNestedBlocksStrictSettings.java"), expected);
```

Comment:
We are not splitting files between individual commits yet.
Please remove the duplicated file and have them point to the same one.